pkg/volume/iscsi/disk_manager.go

```diff
@@ -27,16 +27,16 @@ import (
 
 // Abstract interface to disk operations.
 type diskManager interface {
-	MakeGlobalPDName(disk iscsiDisk) string
+	MakeGlobalPDName(disk iscsiDisk, portal string) string
```

Comment:
`portal` is a known member here in this code path as []string. May be you could use some other name to avoid confusion. Just a thought.